pkg/workloads/tf_train/requirements.oob.txt

```diff
@@ -0,0 +1,11 @@
+numpy==1.15.4
```

Comment:
Do you think it would make sense to have a single shared `requirements.oob.txt` file? Are there differences in what we provide OOB in the different workloads?